modules/exploits/multi/http/splunk_upload_app_exec.rb

```diff
@@ -286,6 +381,48 @@ def do_upload_app(app_name, file_name)
     end
   end
 
+   # version  7.2.x only
+  def do_upload_app_7(app_name, file_name)
+    archive_file_name = ::File.basename(file_name)
+    print_status("Uploading file #{archive_file_name}")
+    file_data = ::File.open(file_name, "rb") { |f| f.read }
```

Comment:
as @bcoles excellently pointed out 7 days ago: https://github.com/rapid7/metasploit-framework/pull/11548#discussion_r264058072
```
file_data = ::File.read(file_name)
```

Reply:
awesome, I  have updated it on both functions